docker/docker.go

```diff
@@ -264,6 +265,31 @@ func WithSharedDirectory(hostPath, containerPath string) ConfigOption {
 }
 
 func withVolume(typ mount.Type, hostPath, containerPath string) ConfigOption {
+	c, err := client.NewEnvClient()
+	if err != nil {
+		logrus.Fatalf("could not create docker client")
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	info, err := c.Info(ctx)
+	if err != nil {
+		logrus.Fatalf("could not get information")
+	}
+
+	isWinHost := info.OSType == "windows" || strings.Contains(
+		strings.ToLower(info.OperatingSystem), "windows")
+	if isWinHost {
+		// For Windows we need to change paths as
```

Comment:
This is actually something that I found when I worked on this issue the first time, but I cannot find the issue anymore in the docker for win repo. 😞